keps/wg-lts/20200122-kubernetes-yearly-support-period.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Kubernetes Yearly Support Period
+authors:
+  - "@imkin"
+  - "@liggit"
+  - "@tpepper"
+  - "@jberkus"
+  - "@youngnick"
+owning-sig: wg-lts
+participating-sigs:
+  - sig-architecture
+  - sig-testing
+  - sig-release
+  - sig-api-machinery
+  - sig-node
+  - sig-cli
+reviewers:
+  - "@quinton_hoole"
+approvers:
+  - TBD
```

Comment:
I would recommend starting with a TLs from owning/participating sigs and let them nominate alternates if they want